extensions/blocks/story/edit.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component, Fragment } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { filter, get, map, pick } from 'lodash';
+import { isBlobURL } from '@wordpress/blob';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { BlockIcon, MediaPlaceholder } from '@wordpress/block-editor';
+import { mediaUpload } from '@wordpress/editor';
+import { DropZone, FormFileUpload, withNotices } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { icon } from '.';
+import Controls from './controls';
+import Story from './story';
+import './editor.scss';
+
+const ALLOWED_MEDIA_TYPES = [ 'image', 'video' ];
```

Comment:
What happens for video files if VideoPress is active?

Reply:
I haven't tested with videopress, but that's something we should do indeed 👍

Reply:
Yup, that would be extremely important for .com sites, especially on Atomic.